portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Workspace.testcase

```diff
@@ -158,6 +158,32 @@ definition {
 		}
 	}
 
+	@description = "Bug: LRAC-8268 | Automation ID: LRAC-11511 | Test Summary: Change friendlyURL for workspace"
+	@priority = "4"
+	test UpdateWorkspaceURLWithFriendlyURL {
+		var currentURL = Navigator.getCurrentURL();
+
+		var extractedAddress = StringUtil.extractFirst("${currentURL}", "/workspace");
+
+		task ("Navigate to Workspace Settings") {
+			ACNavigation.goToSettings();
+
+			ACSettings.goToWorkspace();
+		}
+
+		task ("Set friendly URL") {
+			ACWorkspace.setFriendlyURL(
+				friendlyURL = "${extractedAddress}/workspace/faro-dev-liferay",
+				urlName = "faro-dev-liferay");
+
+			ACUtils.waitForLoading();
+		}
+
+		task ("Assert friendly URL being apply to workspace url") {
+			Navigator.openSpecificURL(url = "${extractedAddress}/workspace/faro-dev-liferay/sites");
+		}
```

Comment:
Hey @calvinctvo, I think after accessing the site you can check that the Sites page is showing after accessing this URL. I think checking the title of the page ("Sites") is good for you.